loadtest/config.go

```diff
@@ -87,7 +87,7 @@ func (ucc *UserControllerConfiguration) IsValid() error {
 		return fmt.Errorf("could not validate configuration: %w", err)
 	}
 	if ucc.Rate < 0 {
-		return errors.New("rate cannot be < 0")
+		return fmt.Errorf("rate cannot be < 0")
```

Comment:
I would keep using the stdlib `errors.New` and keep `fmt.Errorf` only for cases when formatting verbs are required.

Reply:
Sure, somehow I thought we decided to use `fmt` for everything. Will update.

Reply:
We actually do use `fmt.Errorf` quite a lot for simple strings.

```sh
rg "fmt\.Errorf\(\"[^%]*\"\)"
```

If we want to keep consistency we might need a separate PR.

Reply:
Ok, sounds good. :+1:

Reply:
Created https://mattermost.atlassian.net/browse/MM-23914